activerecord-postgis-adapter.gemspec

```diff
@@ -10,16 +10,17 @@ Gem::Specification.new do |spec|
 
   spec.version = ActiveRecord::ConnectionAdapters::PostGIS::VERSION
   spec.authors = ["Daniel Azuma", "Tee Parham"]
```

Comment:
name should be updated and index matched with the emails.

Reply:
these are authors, I do not consider myself as author :/

Reply:
Normally, you keep the old authors and maintainers and prepend active ones.

Reply:
Not according to the specification reference:

> **AUTHORS=(value)
>
> A list of authors for this gem.
> 
> [...]
>
> **Email**
>
> A contact email address (or addresses) for this gem

<sup>

https://guides.rubygems.org/specification-reference/

</sup>

I'd rather not mention myself as author unless I rewrite a significant portion of the gem alone.